Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -66,6 +67,20 @@ def livecheck_formulae(formulae_to_check, args)
       @livecheck_strategy_names.freeze
 
       has_a_newer_upstream_version = false
+
+      if args.json?
```

Comment:
```suggestion
      if args.json? && !args.quiet?
```

Reply:
I had a doubt about this, the help message for `--quiet` says that it should be used to suppress warnings. Would it be okay to use it for this purpose (be non-verbose)?

Reply:
Hmm, that's a good question. I wouldn't want to misuse it, so I'll ask internally and see what the general consensus is.

If this isn't appropriate, then our options would seemingly be to either put the progress bar behind a flag (`--progress`) or add a flag to avoid the progress bar (`--no-progress`). Anyone running the command would specifically have to know about the `no-progress` flag to avoid the progress output when running the command in scripts, so I would probably err on the side of making the progress bar opt-in via a `--progress` flag (if we have to go down this route).

Reply:
I've decided (for now) to implement the `--progress` flag and the `--output-file=` argument. I've finished it locally but the command completions need to be updated, so I'll push after I'm done with those.

Reply:
> Would it be okay to use it for this purpose (be non-verbose)

Yup I think so 👍🏻